client/src/shared/hooks/useEnrollUser.ts

```diff
@@ -60,7 +120,7 @@ const useEnrollUser = (): any => {
     },
     {
       text: courseTitle,
-      url: `/trainer/course/detail/${router.query.id}`
+      url: typeof router.query.id === 'string' ? `/trainer/course/detail/${router.query.id}` : ''
```

Comment:
Because if I remove it Sir a typescript error occur that says "Invalid type "string | string[] | undefined" of template literal expression." and other solution to it is disabling ESLINT. I will try if there are other ways.